ReactWindows/ReactNative.Net46/UIManager/BaseViewManager.cs

```diff
@@ -16,6 +17,22 @@ public abstract class BaseViewManager<TFrameworkElement, TLayoutShadowNode> :
         where TFrameworkElement : FrameworkElement
         where TLayoutShadowNode : LayoutShadowNode
     {
+        /// <summary>
+        /// Instantiates the base class <see cref="BaseViewManager{TFrameworkElement, TReactShadowNode}"/>.
+        /// </summary>
+        protected BaseViewManager()
+        {
+        }
+
+        /// <summary>
+        /// Instantiates the base class <see cref="BaseViewManager{TFrameworkElement, TReactShadowNode}"/>.
+        /// </summary>
+        /// <param name="eventDispatcher">The event dispatcher to associate with this instance</param>
+        protected BaseViewManager(IEventDispatcher eventDispatcher)
```

Comment:
The changed code injects the EventDispatcher into the ViewManager's via property injection in the UIManagerModule, however, the ultimate solution would be to inject this collaborator via the ViewManager's constructor... That will allow us to make the EventDispatcher readonly and prevent any unintentional overriding of that property...